rpm/template.spec.em

```diff
@@ -31,6 +31,9 @@ Source0:        %{name}-%{version}.tar.gz
 %autosetup
 
 %build
+# Needed to bootstrap since the ros_workspace package does not yet exist.
+export PYTHONPATH=@(InstallationPrefix)/lib/python%{python3_version}/site-packages
```

Comment:
This just mimic the Debian patch but I was thinking if we would prefer to preserve the previous `PYTHONPATH` value and just add the new directory.

Do you have at hand a log where I can see the building process in action to check that the values in PYTHONPATH are correct? They seems fine to me, just double checking.

Reply:
Here is the test farm: http://35.174.155.1/job/Ebin_centos_el764__ament_cmake_core__rhel_7_x86_64__binary/8/console#console-section-0

Ping me if you need permission to access the logs - I can't remember if that instance was publicly readable.